docs/sphinx/examples/python/tutorials/noisy_simulations.ipynb

```diff
@@ -99,10 +99,10 @@
     "depolarization_channel = cudaq.DepolarizationChannel(error_probability)\n",
     "\n",
     "# We can also define our own, custom noise channels through\n",
-    "# Kraus Operator's. Here we will define two operators repsenting\n",
+    "# Kraus Operators. Here we will define two operators repsenting\n",
     "# bit flip errors.\n",
     "\n",
-    "# Define the Kraus Error Operator as a complex ndarray.\n",
+    "# Define the Kraus Error Operator as a complex nparray.\n",
```

Comment:
This is not a nparray (numpy array), this is meant to mean an n-dimensional array.
```suggestion
    "# Define the Kraus Error Operator as a complex n-dimensional array.\n",
```